docs/streams/upgrade-guide.html

```diff
@@ -71,6 +71,7 @@ <h1>Upgrade Guide and API Changes</h1>
     </p>
 
     <h3><a id="streams_api_changes_230" href="#streams_api_changes_230">Streams API changes in 2.3.0</a></h3>
+    <p>Version 2.3.0 adds the Suppress operator to the <code>kafka-streams-scala</code> Ktable API.</p>
```

Comment:
Should we also mention the added `EagerBufferConfig` as well?

Reply:
I left it out intentionally; I _think_ that it's unnecessary detail, which would make the guide more cumbersome without adding value.

The reason is that this change doesn't have any effect on the code that people have written or will write. Specifically, the return type of `BufferConfig.emitEarlyWhenFull()` used to be `BufferConfig<?>`, and now it's `EagerBufferConfig extends BufferConfig<?>`, and both are applicable in exactly the same case: `Suppressed.untilTimeLimit(BufferConfig)`, and not `Suppressed.untilWindowCloses(StrictBufferConfig)`.

At the most, it's a change that people may notice in passing, but they can read the javadocs if they're curious. It shouldn't materially affect their lives in any way.